src/styles/block/calendar.scss

```diff
@@ -0,0 +1,59 @@
+.calendar {
+  width: ($size--block * 7)+ 6;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  margin: auto ;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+
+  &__day {
+    width: $size--block;
+    height: $size--block;
+    background-color: #eee;
+    border: 1px solid black;
+    cursor: pointer;
+    position: relative;
+
+    &:hover {
+      transform: translateY(-20px);
+      background-color: #ff3;
+    }
+  }
+
+  @for $index from 1 through 31 {
+    div {
+      &:nth-child(#{$index})::before {
+        content: "#{$index}";
+        font-size: 30px;
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        margin-top: -15px;
+        margin-left: -15px;
```

Comment:
it's not centered
<img width="108" alt="image" src="https://user-images.githubusercontent.com/53611910/226181757-a8eeebc8-6867-4bcf-8e4c-13e97c86affb.png">